refarch-integrations/pom.xml

```diff
@@ -0,0 +1,191 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>3.3.2</version>
+        <relativePath/>
+    </parent>
+
+    <groupId>de.muenchen.refarch</groupId>
+    <artifactId>refarch-integrations</artifactId>
+    <version>1.1.0-SNAPSHOT</version>
+    <packaging>pom</packaging>
+    <name>refarch-integrations</name>
+
+    <modules>
+        <module>refarch-s3-integration</module>
+    </modules>
+
+    <properties>
+        <java.version>21</java.version>
+        <maven.compiler.source>${java.version}</maven.compiler.source>
+        <maven.compiler.target>${java.version}</maven.compiler.target>
+        <maven.compiler.release>${java.version}</maven.compiler.release>
+
+        <jacoco.version>0.8.12</jacoco.version>
+        <maven-surefire-plugin.version>3.3.1</maven-surefire-plugin.version>
+        <spotless-maven-plugin.version>2.34.0</spotless-maven-plugin.version>
+        <itm-java-codeformat.version>1.0.10</itm-java-codeformat.version>
+
+        <spring-cloud.version>2023.0.3</spring-cloud.version>
+        <logstash-logback-encoder.version>8.0</logstash-logback-encoder.version>
+        <mapstruct.version>1.5.5.Final</mapstruct.version>
+        <lombok-mapstruct-binding.version>0.2.0</lombok-mapstruct-binding.version>
+        <springdoc-openapi.version>2.6.0</springdoc-openapi.version>
+    </properties>
```

Comment:
Don't know what you mean. Properties are in same order as in gateway

Reply:
Please take a look at the backend template pom under https://github.com/it-at-m/refarch-templates/blob/main/refarch-backend/pom.xml#L21. I think it would be great to categorize and order them the same way as its done there.